index.html

```diff
@@ -0,0 +1,300 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
+      integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh"
+      crossorigin="anonymous"
+    />
+    <link
+      rel="stylesheet"
+      href="assets/bootstrap/css/bootstrap.min.css"
+      integrity=""
+      crossorigin=""
+    />
```

Comment:
- [ ] Duplication of code. Decide whether use the CDN Bootsrap link or have a local installation of Bootstrap. How to decide the one to keep? if you're planning to customize the Bootstrap framework leave the local version, if you're just gonna use the default Bootstrap classes use only the CDN link. Choose one and don't leave both of them. If you still need to customize some basic stuff or create new classes because of some Bootstrap limits use a custom CSS stylesheet.

Reply:
Done